src/main/java/com/aws/greengrass/logmanager/CloudWatchAttemptLogsProcessor.java

```diff
@@ -61,6 +64,7 @@
     private final Kernel kernel;
     private final DeviceConfiguration deviceConfiguration;
     private static final Logger logger = LogManager.getLogger(CloudWatchAttemptLogsProcessor.class);
+    private static Pattern textTimestampPattern = Pattern.compile("([\\w-:.]+)Z");
```

Comment:
`Instant.parse()` expects ISO 8601, right? Rename this variable to convey that. Also, this regex doesn't support time zone offsets. Please add more test cases for this, I think we're missing a lot here